src/2025h2/bridge-between-incrementals.md

```diff
@@ -0,0 +1,99 @@
+# Bridge between incremental caches for all tools
+
+| Metadata         |                                                                                  |
+|:-----------------|----------------------------------------------------------------------------------|
+| Point of contact | @blyxyas                                                                         |
+| Teams            | (probably WG-incremental but waiting approval by them)                           |
+| Task owners      | @blyxyas                                                                          |
+| Status           | Proposed                                                                         |
+| Tracking issue   |                                                                                  |
+| Zulip channel    |                                                                                  |
+
+## Summary
+
+The goal of this project goal is to make a compatibility layer to share incremental compilation information between
+`cargo check`, `cargo build`, and any other external tool that might benefit from sharing incremental information
+such as `cargo clippy`.
+
+## Motivation
```

Comment:
Unsure where this might fit but our issue for this is https://github.com/rust-lang/cargo/issues/3501